moabb/evaluations/evaluations.py

```diff
@@ -16,13 +22,46 @@
 
 
 class WithinSessionEvaluation(BaseEvaluation):
-    """Within session evaluation
-
-    returns Score computed within each recording session
-
-    """
-
-    def evaluate(self, dataset, pipelines):
+    """Within Session evaluation."""
+
+    def __init__(
+        self,
+        n_perms: Union[int, np.ndarray] = 1,
```

Comment:
Good point, I had the `self.calculate_learning_curve` based on checking whether `n_perms == 1` before but noticed that it does not make much sense. I could even initialize it to None to emphasize the fact that both parameters are optional and only needed if you want a learning curve. Otherwise only the classical `WithinSessionEvaluation` is run.